flink-cyber/cyber-jobs/src/main/resources/examples/pipelines/basic/triage/load_majestic_million.properties.template

```diff
@@ -0,0 +1,12 @@
+enrichment.type=majestic_million
+enrichment.config.file=generated/basic/triage/enrichments-storage.json
```

Comment:
The scripts only replace BRANCH but it would be easier to rename the pipelines if it replaced PIPELINE as well.  In the current state it's hard to rename a pipeline.